pkg/utils/args/cidr.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package args
+
+import "net"
+
+// Cidr implements the flag.Value interface and allows to parse strings in CIDR format lists
+// in the form: "x.x.x.x/y".
+type Cidr struct {
```

Comment:
Acronyms should have consistent case (CIDR in this case): https://github.com/golang/go/wiki/CodeReviewComments#initialisms